server/api/activity/__tests__/activity.spec.js

```diff
@@ -15,6 +15,9 @@ import acts from './activity.fixture.js'
 test.before('before connect to database', async (t) => {
   t.context.memMongo = new MemoryMongo()
   await t.context.memMongo.start()
+  t.context.people = await Person.create(people).catch((err) => `Unable to create people: ${err}`)
+  t.context.orgs = await Organisation.create(orgs).catch((err) => `Unable to create organisations: ${err}`)
```

Comment:
These `.catch` calls will swallow errors and allow tests to run even though fixtures haven't been loaded.

Removing the `.catch` calls and just letting any exceptions get thrown will give much faster feedback to the person running the tests - which is preferable in my opinion

Reply:
Agreed - but this pattern is repeated across almost all tests. The only time you'll see the error is when the fixture is invalid and throws a mongo error. That will be printed in the output.  I'll leave for now and add a new ticket.

Reply:
https://voluntarily.atlassian.net/browse/VP-1335